lib/controllers/game_controller.cpp

```diff
@@ -203,6 +218,7 @@ void GameController::setupInput() {
     inputMapping[1][InputKeyCode::EKey_pagedown] = PlayerInput::SPEED_DOWN;
     inputMapping[1][InputKeyCode::EKey_pageup] = PlayerInput::SPEED_UP;
     inputMapping[1][InputKeyCode::EKey_home] = PlayerInput::SPEED_RESET;
+    inputMapping[1][InputKeyCode::EKey_escape] = PlayerInput::PAUSE;
```

Comment:
You are not able to pause the game if Player 1 does not exist.

Reply:
With the keyboard*

Reply:
Shall I change this for the whole line up above as the other function will not be available either?